constantvariables.py

```diff
@@ -6,4 +6,16 @@
 
 VERIFIED_MEMBER_ID = 1120501110407434301 # Replace with the actual ID of the role you want for the greetings message
 
-WELCOME_REACT_EMOJI = "👍" # Replace with the emoji you want to use for your welcome message react emoji
\ No newline at end of file
+WELCOME_REACT_EMOJI = "👍" # Replace with the emoji you want to use for your welcome message react emoji
+
+GAME_CHANNELS = { # Key must be video game name and value must be maximum number of players that can play in a full squad
+    "Valorant": 5,
+    "League of Legeneds": 5,
+    "Rocket League": 3
+}
+
+DIFFERENT_NAME_CHANNELS = { # REQUIRED: Must have lounge, study, and cinema channels. You can name them whatever you want but make sure to replace the channel IDs with your server's
+    "Lounge": 1132567562014691418,
```

Comment:
We're using a lot of channel IDs in our constant variables which is great. What is the best way to figure out the channel IDs of each server? Is there a script that we can run or put in our TODO so that people know how to do this? Can we make a wiki page for it as a HOW-TO?

Reply:
Channel ID has to be provided by the developer.